indico/modules/events/registration/placeholders/registrations.py

```diff
@@ -31,6 +32,26 @@ def render(cls, regform, registration):
         return registration.last_name
 
 
+class PicturePlaceholder(Placeholder):
+    name = 'picture'
+    description = _('Picture of the person')
+    max_size = '250px'
+
+    @classmethod
+    def render(cls, regform, registration):
+        picture_data = registration.get_personal_data_picture()
+        if not picture_data:
+            return ''
+        style = f'"max-height: {cls.max_size}; max-width: {cls.max_size};'
+        style += ' white-space: normal; padding-bottom: 10px; object-fit: cover;"'
+        if request.endpoint == 'event_registration.email_registrants_preview':
+            url = url_for('event_registration.registration_picture', picture_data.locator.file,
```

Comment:
I think this should use the `manage_registration_file` endpoint since we're in a management context here.